### PR TITLE
Fix outdated links on the landing page and minting details

### PR DESCRIPTION
**Pull request by samaradel**
### Description

Fix outdated links on the landing page and minting details as per discussion with @samtaggart 

### Changes

image

- ```learn about the grid``` link ----> [manual](https://manual.grid.tf/)
- ```minting support``` link ----> [threefold.io](https://threefold.io/)

### Related Issues

- #2359
- #2363

### Checklist

- [ ] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by zaelgohary — APPROVED** [review]

File: packages/playground/src/components/logged_in_landing.vue [hunk 1]
```diff
@@ -74,16 +74,9 @@
...
-                  color="secondary"
-                  variant="outlined"
-                  class="mr-2"
-                  target="_blank"
-                  href="https://www.threefold.io/grid/"
-                >
+                <v-btn color="secondary" variant="outlined" class="mr-2" target="_blank" href="https://manual.grid.tf/">
                   Learn about the grid
                 </v-btn>
-                <v-btn color="primary" target="_blank" href="https://www.threefold.io/build/"> Use The Grid </v-btn>
```
Comment (amiraabouhadid) [inline packages/playground/src/components/logged_in_landing.vue:86]: why did you remove this button?

Comment (samaradel) [reply packages/playground/src/components/logged_in_landing.vue:86]: was @samtaggart suggestion

**Review by maayarosama — COMMENTED** [review]
In minting page the following 


navigates to FAQ page

**Comment by samaradel** [comment]
@samtaggart the new link does not exist in the manual. can you confirm, please

**Comment by samaradel** [comment]
@maayarosama good catch, thanks :)

**Comment by samtaggart** [comment]
Hello am I still needed here? Looks like the PR was merged.
